src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -103,16 +107,20 @@ class WorkspacePageWithSections extends React.Component {
                     onBackButtonPress={() => Navigation.navigate(ROUTES.getWorkspaceInitialRoute(policyID))}
                     onCloseButtonPress={() => Navigation.dismissModal()}
                 />
-                <ScrollView
-                    keyboardShouldPersistTaps="handled"
-                    style={[styles.settingsPageBackground, styles.flex1, styles.w100]}
-                >
-                    <View style={[styles.w100, styles.flex1]}>
-
-                        {this.props.children(hasVBA, policyID, isUsingECard)}
-
-                    </View>
-                </ScrollView>
+                {this.props.shouldUseScrollView
```

Comment:
I think `this.props.shouldUseScrollView` could use a better name. 
It should indicate _why_ we want to use scroll view.